common/live/stg/infra/dev/tyler/terraform.tfvars

```diff
@@ -0,0 +1,18 @@
+# ↓ Module metadata
```

Comment:
Creation of this file is reverted in #443. It appears in this PR because it was convenient to leave it here during debugging. It will disappear once #443 is merged.